drivers/ethernet/Kconfig.mcux

```diff
@@ -28,6 +28,13 @@ config ETH_MCUX_RMII_EXT_CLK
 	  Setting this option will configure MCUX clock block to feed RMII
 	  reference clock from external source (ENET_1588_CLKIN)
 
+config ETH_MCUX_NO_PHY_SMI
```

Comment:
merge this commit with the previous one.

Reply:
Sorry, but I don't see the exact commit SHAs.
Which two commits do you recommend to merge?